docs/src/snap/howto/cis-hardening.md

```diff
@@ -0,0 +1,270 @@
+# CIS compliance 
+
+CIS Hardening refers to the process of implementing security configurations that
+align with the benchmarks set by the [Center for Internet Security (CIS)][]. The
+open source tool [Kube-bench][] is designed to automatically check whether
+your Kubernetes clusters are configured according to the 
+[CIS Kubernetes Benchmark][]. This guide covers how to:
+
+- [Install and configure kube-bench](#install-kube-bench)
+- [Harden your deployments](#harden-your-deployments)
+- [Audit your deployments](#audit-your-deployments)
+
+## What you'll need
+
+- A bootstrapped {{product}} cluster (see the [Getting Started]
+[getting-started-guide] guide)
+- Root or sudo access to the machine
+
+## Install kube-bench 
+
+Download the latest [kube-bench release][] on your Kubernetes nodes. Make sure 
+to select the appropriate binary version.
+
+For example, to download the Linux binary, use the following command. Replace 
+x.x by the version listed in the releases page.
+
+```sh 
+mkdir kube-bench
+
+cd kube-bench
+
+curl -L https://github.com/aquasecurity/kube-bench/releases/download/v0.x.x/kube-bench_0.x.x_linux_amd64.tar.gz -o kube-bench_0.x.x_linux_amd64.tar.gz
+```
+
+Extract the downloaded tarball and move the binary to a directory in your PATH:
+
+```sh
```

Comment:
agin, if you have a variable, you can make this nicer for users

Reply:
Used same var as in previous snippet